x-pack/dockerlogbeat/docs/usage.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[log-driver-usage-examples]]
+== {log-driver} usage examples
+
+++++
+<titleabbrev>Usage examples</titleabbrev>
+++++
+
+experimental[]
+
+TODO: Add examples that show commands and daemon file. Some possibilities: 
```

Comment:
@fearful-symmetry Can you suggest some usage examples that we can show in this topic? I think we should show the command and daemon file. WDYT? If you think the examples in the installation docs are good enough for now, I can comment out this file.

BTW, the list I show here simply corresponds to settings that we illustrate with examples in the Beats docs. Would be best to come up with a list of common configurations that we expect users will want to use.

Reply:
Thanks...I think these are pretty similar to what I already show in the install instructions. Maybe I will put the examples in files that I can include in both places for users who don't read carefully. :-) If we don't plan/need to provide more complex examples, then I will just collapse this topic onto the same page as the option descriptions.